File: new test/solution3.go
package main

import "fmt"

var testInput = map[int]int{
    1:1,
}

var testA = []int{1,2}
var testB = []int{1,2}

func main()  {
    for i := range testA {
        fmt.Println(solution(testA[i], testB[i]))
    }
}

func solution(a int, b int) int {
    return a + b
}

File: new test/solution3.js
function main () {
    let testA = [1,2,3]
    let testB = [1,3,4]

    for (let i = 0; i < testA.length ; i++) {
        console.log(solution(testA[i], testB[i]))
    }

}

function solution(testA, testB) {
    let c = new Map()
    c.get(1)
    return testA + testB
}

main()